### ISSUE TITLE
Incorrect type for breakpoints 

### ISSUE THREAD

**Issue by StephanCodes** [jkhaui/react-glidejs#23]
Hi, I just started using this library and I'm very grateful for it. I noticed, however, that the type for the breakpoints appears to be incorrect. It is currently:
```ts
declare type Breakpoints = Record<number, object[]>;
```
But I believe it should be:
```ts
declare type Breakpoints = Record<number, object>;
```

I based this on the documentation of GlideJS itself: https://glidejs.com/docs/options/#breakpoints
Thanks again!

**Comment by arthurmmedeiros** [comment]
Were you able to make it work anyway? 

It is not working to me, even if I provide the expected object.

**Comment by StephanCodes** [comment]
Sorry, I eventually switched to another solution entirely. If I remember correctly, it did work after changing the type manually in the library. 🤔

**Comment by arthurmmedeiros** [comment]
Could you share how you managed to change the type manually in the library? Was it by overriding the type somehow?

**Comment by StephanCodes** [comment]
Yeah, I did it in the ugliest way, by directly editing the file in my `node_modules` folder. 😅
